packages/bits/src/constants/unit-conversion.constants.ts

```diff
@@ -20,7 +20,7 @@ export enum UnitBase {
  * Unit display values used in unit conversion
  */
 export const unitConversionConstants: IUnitConversionConstants = {
-    generic: ["", "k", "M", "B", "T", "Qa", "Qi", "Sx", "Sp"],
+    generic: ["", "K", "M", "B", "T", "Qa", "Qi", "Sx", "Sp"],
     bytes: ["B", "KB", "MB", "GB", "TB", "PB", "EB", "ZB", "YB"],
     bytesPerSecond: ["Bps", "kBps", "MBps", "GBps", "TBps", "PBps", "EBps", "ZBps", "YBps"],
```

Comment:
Not related, but this seems like a strange design considering the "bps" and "Bps" units are always the same. The only thing that matters here are the prefixes.